### PR TITLE
HL-181-2

### PR DESCRIPTION
**Pull request by OptimusOpus**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-181)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1271,7 +1271,7 @@ app.post("/bets/sign", async (req, res) => {
     // Is there a result for this market?
     const marketResult = await marketOracleContract.getResult(marketId);
     // 0x000... equates to no result
-    const resultAdded = marketResult !== ethers.constants.HashZero;
+    const resultAdded = marketResult !== "0x00000000000000000000000000000000";
```
Comment (brettvniekerk) [inline src/index.js:1274]: what is the difference?

Comment (bazmatic) [reply src/index.js:1274]: Why have we removed the constant?

Comment (bazmatic) [inline src/index.js:1274]:
```suggestion
   const resultAdded = marketResult !== ethers.constants.HashZero;
```

Comment (bazmatic) [reply src/index.js:1274]: Oh you're making it bytes16

Comment (brettvniekerk) [reply src/index.js:1274]: ah ok that makes sense

**Review by brettvniekerk — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**